uber/config.py

```diff
@@ -222,28 +221,34 @@ def ATTENDEE_BADGES_SOLD(self):
     @dynamic
     def BADGES_SOLD(self):
         from uber.models import Session, Attendee, Group
-        with Session() as session:
-            attendees = session.query(Attendee)
-            individuals = attendees.filter(or_(
-                Attendee.paid == self.HAS_PAID,
-                Attendee.paid == self.REFUNDED)
-            ).filter(Attendee.badge_status == self.COMPLETED_STATUS).count()
+        if self.BADGES_SOLD_ESTIMATE_ENABLED:
+            with Session() as session:
+                attendee_count = int(session.execute(
+                    "SELECT reltuples AS count FROM pg_class WHERE relname = 'attendee'").scalar())
```

Comment:
Whoa, is this our first ever instance of pure SQL inside the codebase? 😮

Reply:
Maybe??? It will break on any non-Postgres database 😅

Reply:
I think realistically we don't have a lot of potential events going "I would love this, but it HAS to use MySQL" anyway :P

Reply:
Meh, they could just set `BADGES_SOLD_ESTIMATE_ENABLED = False` anyway!